.travis.yml

```diff
@@ -52,14 +52,15 @@ jobs:
       os: linux
       script:
         - make
+        - sudo make install
```

Comment:
`sudo` is introduced here. Do we already require `sudo` before the change?

Reply:
Yeah. See the `run-critest.sh` command below. :)

I also need to add `sudo` for the ginkgo copy.